React/Views/RCTNavItemManager.m

```diff
@@ -39,4 +39,21 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(rightButtonIcon, UIImage)
 RCT_EXPORT_VIEW_PROPERTY(rightButtonTitle, NSString)
 
+RCT_CUSTOM_VIEW_PROPERTY(titleFontSize, CGFloat, RCTNavItem)
+{
+  view.titleFont = [RCTConvert UIFont:view.titleFont withSize:json ?: @(defaultView.titleFont.pointSize)];
+}
+RCT_CUSTOM_VIEW_PROPERTY(titleFontWeight, NSString, RCTNavItem)
+{
+  view.titleFont = [RCTConvert UIFont:view.titleFont withWeight:json ?: @(UIFontWeightRegular)];
+}
+RCT_CUSTOM_VIEW_PROPERTY(titleFontStyle, NSString, RCTNavItem)
+{
+  view.titleFont = [RCTConvert UIFont:view.titleFont withStyle:json ?: @(NO)];
```

Comment:
My bad was trying to figure out if I could pass in a value using defaultView.titleFont.___ but there wasn't anything there. The RCTConvert takes in @NO just fine though. @"normal" gets converted into @NO eventually.